Pipfile

```diff
@@ -57,7 +57,7 @@ pandas-stubs = "*"
 playwright = "*"
 pre-commit = "*"
 pyopenssl = "*"
-pytest = "*"
+pytest = "==7.4.3"
```

Comment:
Is this necessary?

Reply:
It's not really necessary, but I think having no fixed version, or at least no fixed major version can be bad. But I can move back to the `*` notation.